packages/libs/react-components/src/components/Input/Input.tsx

```diff
@@ -16,8 +16,8 @@ export interface IInputProps
   > {
   as?: 'input';
   leadingText?: string;
-  leftPanel?: typeof SystemIcons[keyof typeof SystemIcons];
-  rightPanel?: typeof SystemIcons[keyof typeof SystemIcons];
+  leftPanel?: (typeof SystemIcons)[keyof typeof SystemIcons];
```

Comment:
Comment left in other PR as well: Do you know what is causing this formatting change? Maybe it's a global prettier file? The change isn't semantically wrong, but I've noticed when we've merged this change in the past, it gets changed back by other PRs so we want to avoid these formatting inconsistencies if possible